conda/environments/bench_ann_cuda-118_arch-x86_64.yaml

```diff
@@ -34,6 +34,7 @@ dependencies:
 - nccl>=2.9.9
 - ninja
 - nlohmann_json>=3.11.2
+- pyyaml
```

Comment:
Matplotlib doesn't seem to be getting installed as a dependency of the `libraft-ann-bench` conda package.

Reply:
Yes, it's part of the bench-ann environment file. It should not be a dependency of the libraft-ann-bench conda package because that package is just the executables